modules/URL/ConferenceUrl.js

```diff
@@ -0,0 +1,73 @@
+
+import { redirect } from '../util/helpers';
+
+/**
+ * Stores the original conference room URL with all parameters.
+ * @type {string}
+ */
+let originalURL;
+
+/**
+ * A simplified version of the conference URL stripped out of the parameters
+ * which should be used for sending invites.
+ * @type {string}
+ */
+let inviteURL;
+
+/**
+ * The modules stores information about the URL used to start the conference and
+ * provides utility methods for dealing with conference URL and reloads.
+ */
+export default {
```

Comment:
I don't see how imports are so bad, but global variables so cool. But agree that this could be a class as well and don't mind changing it.